src/ut/cache_test.cpp

```diff
@@ -133,9 +170,9 @@ class CacheRequestTest : public CacheInitializationTest
   {
     sem_init(&_sem, 0, 0);
 
-    // By default the cache just serves up the mock client each time.
-    EXPECT_CALL(_cache, get_client()).WillRepeatedly(Return(&_client));
-    EXPECT_CALL(_cache, release_client()).WillRepeatedly(Return());
+    // In some tests we check for success() explicitly, but  this removes the
+    // warning for those where we don't
+    EXPECT_CALL(_resolver, success(_)).Times(testing::AnyNumber());
```

Comment:
But you basically have an expect call in every testcase anyway, so do you actually need this?